### ISSUE TITLE
cancel option etc

### ISSUE THREAD

**Issue by oberon-manjaro** [Chrysostomus/brandr#1]
Please have a look. Something is not right here.
Option 2 gives me

```
1) eDP1
2) HDMI1
3) DP1
4) proceed
5) cancel
```

Hitting 4 does:

```
Choose monitor to operate on: 4
sed: -e expression #1, char 0: no previous regular expression
sed: -e expression #1, char 0: no previous regular expression
1) proceed
2) cancel
```

and hitting 5 instead of 4, so should be "cancel", does:

```
Choose monitor to operate on: 5
sed: -e expression #1, char 0: no previous regular expression
sed: -e expression #1, char 0: no previous regular expression
1) proceed
2) cancel
Choose new resolution:
```

Similar situation also with other menu entries...

**Comment by Chrysostomus** [comment]
I made some adjustments. Does it work better now?

**Comment by Chrysostomus** [comment]
cancel in option 4 is still problematic (ugly), but kind of gets the job done.

**Comment by oberon-manjaro** [comment]
option 4 gives me just

```
Choose monitor to operate on >
 1/1
> eDP1
```

Here I have no option to cancel.
Now I type 1 and Enter. Brings me back to a selection, including 6) cancel.
But selecting 6 presents me with

```
Relative to which monitor? >
...
```

Something's not right here, yet ;)

**Comment by Chrysostomus** [comment]
Yeah, it was the best I could manage for now. Selecting cancel does not cancel the command, but instead runs it i a way that does not do anything. Maybe the easiest option would be to hide the error message conditionally?

**Comment by oberon-manjaro** [comment]
I don't know enought about PS3. What about this "proceed" and "cancel"? Are that predefined default values? No way to use '0' instead to return to the menu?

**Comment by Chrysostomus** [comment]
Nope, builtin select starts counting from 1 and assigns number to each menu item. I can have 0 as menuitem, but then it would be 
1)0
And you would still have to press 1 to choose it. So no, can't do that with select. The mainmenu uses builtin read, but that requires hardcoded menu entries. Or some creativity, but I have that in short supply.

**Comment by oberon-manjaro** [comment]
I see ;D Having to press 1 in order to say 0 is certainly not ideal ;D

**Comment by Chrysostomus** [comment]
There is improvement now. You still have to choose the last option even if you choose cancel in the middle, but there is relevant cancellation message and no errors.

**Comment by oberon-manjaro** [comment]
cool! :) I will check later.
